src/Slider.jsx

```diff
@@ -625,7 +625,7 @@ class Rheostat extends React.Component {
       // eslint-disable-next-line jsx-a11y/no-static-element-interactions
       <div
         className={className}
-        ref="rheostat"
+        ref={(slider) => { this.rheostat = slider; }}
```

Comment:
This actually needs to use a constructor-bound instance method, otherwise react will call it alternating with null and the ref.